roles/edpm_telemetry/tasks/enable_logging.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy rsyslog configuration
+  become: true
+  ansible.builtin.copy:
+    src: "{{ edpm_telemetry_config_src }}/10-telemetry.conf"
+    dest: /etc/rsyslog.d/10-telemetry.conf
+    mode: "0644"
+    remote_src: "{{ telemetry_test | default('false') }}"
+  tags:
+    - edpm_logging
+
+- name: Restart rsyslog
+  become: true
+  ansible.builtin.systemd:
+    state: restarted
+    name: rsyslog.service
+  tags:
```

Comment:
Since this task file is called only by the above mentioned playbook, I'd change this task to be an handler and I'd add the directive:

` notify: Restart rsyslog` 

to the copy task above.

Reply:
This is a very good idea, but as we are using import_tasks to use the tasks in the role, I fail to see where would the handlers be defined. I also can't found an example of handlers usage in the whole edpm-ansible.

Could you please point me in the right direction here?